codegen.yml

```diff
@@ -1,7 +1,6 @@
 overrideExisting: true
 schema: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3'
-documents: 'src/**/!(*.d).{ts,tsx}'
 generates:
-  ./src/graphql/schema/schema.graphql:
+  ./src/graphql/thegraph/schema/schema.graphql:
```

Comment:
`/thegraph/schema/schema.graphql` feels redundant. Is there something else in this directory, or can you shorten it to `/thegraph/schema.graphql`?